project/Release.scala

```diff
@@ -6,11 +6,11 @@ package akka
 import sbt._
 import sbt.Keys._
 import java.io.File
-import com.typesafe.sbt.site.SphinxSupport.{ generate, Sphinx }
```

Comment:
do we have any more sphinx left, can we remove all of it?
We can just remove all of the contrib docs https://github.com/akka/akka/tree/master/akka-contrib/docs
It's deprecated.